client/src/config/app/locale/en/json/reader.json

```diff
@@ -81,6 +82,8 @@
       "reader_appearance": "Reader Appearance"
     },
     "my_private_annotations": "My Private $t(glossary.annotation_title_case_other)",
-    "my_public_annotations": "My Public $t(glossary.annotation_title_case_other)"
+    "my_public_annotations": "My Public $t(glossary.annotation_title_case_other)",
+    "full_text": "Full Text",
+    "section_description": "Start reading this text on {{append}}"
```

Comment:
I might use a variable name here that would be more descriptive for the translator? `appName` or something? This isn't you; this is case where the old component uses an uninformative variable name.